### ISSUE TITLE
Records can't select templates

### ISSUE THREAD

**Issue by GawainLynch** [bolt/bolt#5119]
![image](https://cloud.githubusercontent.com/assets/1427081/14276829/ad7693a4-fb20-11e5-8c18-1c0a9319acdb.png)

**Comment by bobdenotter** [comment]
Can you post the used contenttype.yml? I can't reproduce this, yet. 

Screenshot: 

![screen shot 2016-04-05 at 15 01 37](https://cloud.githubusercontent.com/assets/1833361/14282590/653d8614-fb3f-11e5-8156-885962c340e6.png)

**Comment by bobdenotter** [comment]
Nevermind, found it. :-)

**Comment by bobdenotter** [comment]
It breaks if your theme.yml has 'named' extrafield templates.. 

ie. this is fine: 

```
templatefields:
    extrafields.twig:
        section_1:
            type: text
            label: Section 1
        section_2:
            type: html
            label: Section 2
        image:
            type: image
    anotherextrafields.twig:
        markdown:
            type: markdown
            label: A markdown field
        geolocation:
            type: geolocation
            label: A geolocation field
        image:
            type: image
```

But add this, and it breaks: 

```
# There can be defined the templates for the template chooser
templateselect:
    templates:
         - name: 'Template with extra fields'
           filename: 'extrafields.twig'
         - name: 'Another template with extra fields'
           filename: 'anotherextrafields.twig'
```

So, this is related to: #5068

Should we just kill off 'named extrafield templates' ? I think it's buggy and confusing as heck.

**Comment by GawainLynch** [comment]
Not sure I care on approach too much, but as it causes data loss

**Comment by bobdenotter** [comment]
I vote :fire: :fire: :fire:

**Comment by bobdenotter** [comment]
Fixed by #5217
